config/checkstyle_checks.xml

```diff
@@ -115,7 +115,7 @@
   </module>
   <module name="RegexpOnFilename">
       <property name="folderPattern" value="[\\/]src[\\/]it[\\/]java[\\/]"/>
-      <property name="fileNamePattern" value="^((\w+Test)|(Base\w+))\.java$"/>
+      <property name="fileNamePattern" value="^((\w+Test)|(Base\w+)|(package-info))\.java$"/>
```

Comment:
good catch by checkstyle....
We need to suppress MissingPackageInfo in tests, as "package-info" file is required only for javadoc generation, but we do not generate javadoc for tests at all - http://checkstyle.sourceforge.net/apidocs/index.html .
Please use inspection MissingPackageInfo only for "Production" scope in IDEA.
Sorry that I mislead you, system is already smarter than I.